docs/direct/api/v1.md

```diff
@@ -1485,490 +1862,2635 @@ The properties of each endpoint type are listed below.
+  "faxEnabled": false
+}
+```
+ </TabItem>
+</Tabs>
+
+#### Add a new Phone Number
+
+`POST /customers/{customerId}/phonenumbers`
+
+To add an availablenumber to a customer account please use a POST as per the below.
+NB: This number should be available in the simwood inventory. you can check Available numbers api , to see available numbers.
+
+<Tabs>
+  <TabItem value="json" label="Example Request">
+
+
+```
+POST   https://pbx.sipcentric.com/api/v1/customers/25/phonenumbers
+
+{
+  "createFrom":"4-44-113-4037921",
+  "type":"did"
+}
+```
+
+  </TabItem>
+  <TabItem  value="result" label="Example Response">
+
+```json
+HTTP/1.1 200 OK
+
+{
+  "type": "did",
+  "id": "94854",
+  "uri": "https://pbx.sipcentric.com/api/v1/customers/25/phonenumbers/94854",
+  "parent": "https://pbx.sipcentric.com/api/v1/customers/25",
+  "created": "2023-05-12T14:11:06Z",
+  "number": "01134037921",
+  "formatted": "0113 403 7921",
+  "smsCapable": false,
+  "faxEnabled": false,
+  "links": {
+    "routingRules": "https://pbx.sipcentric.com/api/v1/customers/25/phonenumbers/94854/routingrules"
+  }
+}
+```
+ </TabItem>
+</Tabs>
+
+
+#### Linking Phone Number to extension
+
+`PUT /customers/{customerId}/phonenumbers/{phonenumberId}`
+
+To link a phonenumber in a customer's account to an extension inside the customer account please use a PUT as per the example below.
+"id" in the body is the ID of the phone number, and "destination" is the ID of the extension we want to link to.
+
+NB: This phone number should already be added to the account.  Please see , adding Phone numbers.
+
+<Tabs>
+  <TabItem value="json" label="Example Request">
+
+
+```json 
+PUT https://pbx.sipcentric.com/api/v1/customers/{customerId}/phonenumbers/{phonenumberId}/
+
+{
+  "type": "did",
+  "id": "73224",
+  "uri": "https://pbx.sipcentric.com/api/v1/customers/25/phonenumbers/{phonenumberId}",
+  "parent": "https://pbx.sipcentric.com/api/v1/customers/25",
+  "created": "2022-01-06T11:12:46Z",
+  "number": "01134031453",
+  "formatted": "0113 403 1453",
+  "smsCapable": false,
+  "faxEnabled": false,
+  "links": {
+    "routingRules": "https://pbx.sipcentric.com/api/v1/customers/25/phonenumbers/{phonenumberId}/routingrules"
+  },
+  "destination":"89937"
+}
+```
+
+  </TabItem>
+  <TabItem  value="result" label="Example Response">
+
+```json
+HTTP/1.1 200 OK
+
+{
+    "type": "did",
+    "id": "94854",
+    "uri": "https://pbx.sipcentric.com/api/v1/customers/25/phonenumbers/94854",
+    "parent": "https://pbx.sipcentric.com/api/v1/customers/25",
+    "created": "2023-05-12T14:11:06Z",
+    "number": "01134037921",
+    "formatted": "0113 403 7921",
+    "destination": "https://pbx.sipcentric.com/api/v1/customers/25/endpoints/89937",
+    "smsCapable": false,
+    "faxEnabled": false,
+    "links": {
+        "routingRules": "https://pbx.sipcentric.com/api/v1/customers/25/phonenumbers/94854/routingrules"
+    }
+}
+```
+
+ </TabItem>
+</Tabs>
+
+#### UN-Linking Phone Number and extension
```

Comment:
Do we need examples like this? Could just be one for setting a destination endpoint and a note about how to un-set. This feels too specific for extensions, where should be "endpoints".